core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.ts

```diff
@@ -69,8 +73,9 @@ export class CkEditorCopyContentService {
         containedWidgetTagName = currentTagName;
         break;
       }
-
-      if (currentElement.parentElement.tagName === this.OUTPUT_VIEW_TAG_NAME) {
+      // Prevents reference errors if the element clicked on is itself the
+      // OUTPUT_VIEW_TAG_NAME (i.e, ANGULAR-HTML-BIND).
+      if (this.isElementOrParentElementOutputView(currentElement)) {
```

Comment:
You're right, I don't think we need this function in the first place. I only added this because I faced a "Cannot find property tagName of null" error a few times while clicking on the empty part of the text box within the translation modal.

Reply:
>  I faced a "Cannot find property tagName of null" error a few times while clicking on the empty part of the text box within the translation modal.

Is it still reproducible?

Reply:
Yes it unfortunately is, I guess it's better to track that issue separately.

Reply:
Can you please create an issue for the same?

Reply:
I'll do it right now.

Reply:
Tagged you.